### PR TITLE
Keybind path

### PR DESCRIPTION
**Pull request by myryk31415**
I added the option to change the path to the keybind.conf (since mine is not at the same place).
Also included a check so people get a notification if they entered an incorrect path. `~` in the path now works as well.

The keybindList and keybindListfunc probably be done simpler I just didn't know how.

### CODE REVIEW

File: .config/ags/modules/.configuration/user_options.js [hunk 1]
```diff
@@ -56,6 +56,9 @@ let configOptions = {
             'default': "auto",
         },
     },
+    'configPaths': {
```
Comment (end-4) [inline .config/ags/modules/.configuration/user_options.js]: idk why but i don't like this section name

File: .config/ags/modules/cheatsheet/keybinds.js [hunk 2]
```diff
@@ -5,13 +5,22 @@ import Widget from "resource:///com/github/Aylur/ags/widget.js";
...
-const HYPRLAND_KEYBIND_CONFIG_FILE = `${GLib.get_user_config_dir()}/hypr/hyprland/keybinds.conf`;
+const HYPRLAND_KEYBIND_CONFIG_FILE = userOptions.configPaths.keybinds ?
+    userOptions.configPaths.keybinds : `${GLib.get_user_config_dir()}/hypr/hyprland/keybinds.conf`;
 const KEYBIND_SECTIONS_PER_PAGE = 3;
-const keybindList = JSON.parse(
-    Utils.exec(
-        `${App.configDir}/scripts/hyprland/get_keybinds.py --path ${HYPRLAND_KEYBIND_CONFIG_FILE}`,
-    ),
-);
+const keybindListfunc = () => {
```
Comment (end-4) [inline .config/ags/modules/cheatsheet/keybinds.js]: ugh let's call this `getKeybindList`?

File: .config/ags/modules/.configuration/user_options.js (hunk 1 above)
Comment (myryk31415) [reply .config/ags/modules/.configuration/user_options.js]: what would you name it? I tried to make it a group where other custom paths fit as well if ever needed

**Comment by myryk31415** [comment]
`'customPaths'`, `'fileLocations'`, `'filePaths'`, `'customDirectorys'` or rather something completely different?

**Pull request merged**